client/dashboard/sites/settings-static-file-404/summary.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { notFound } from '@wordpress/icons';
+import RouterLinkSummaryButton from '../../components/router-link-summary-button';
+import { canSetStaticFile404Handling } from './index';
+import type { Site } from '../../data/types';
+
+export default function StaticFile404SettingsSummary( { site }: { site: Site } ) {
+	if ( ! canSetStaticFile404Handling( site ) ) {
```

Comment:
How do we expect users to get to the setting page to see the callout?

Reply:
Good catch, this was leftover code before adding the Callout. Updated in 199c514e416be91a8ddb988d1a0a3b30577d3369.